packages/css/src/plugins/atomicify-rules.ts

```diff
@@ -8,8 +8,21 @@ interface PluginOpts {
   selectors?: string[];
   atRule?: string;
   parentNode?: Container;
+  hashPrefix?: string;
 }
 
+/**
+ * Returns true if a given string is a valid CSS identifier
+ *
+ * @param value the value to test
+ * @returns `true` if given value is valid, `false` if not
+ *
+ */
+const isCssIdentifierValid = (value: string): boolean => {
+  const validCssIdentifierRegex = /^[a-zA-Z\-][a-zA-Z\-_0-9]*$/;
```

Comment:
where did you find this regex? it looks reasonable but it doesn't match with what i could easily find online e.g. https://stackoverflow.com/a/449000

Reply:
I made it myself as it was supposed to be in the middle of an already existing identifier. It's actually not 100% now that I look at it; 

That said, since it's now being used in the hash and not as a standalone string it could be pretty much anything but I think it makes sense to continue to only accept identifiers to prevent any potential unwanted side effects. I'll refactor it